CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -80,40 +87,58 @@ public bool TryCallMethod(long objectId, string name, object[] parameters, out o
                 return false;
             }
 
-            var method = obj.Methods.FirstOrDefault(p => p.JavascriptName == name);
-            if (method == null)
+            var overloads = obj.Methods.Where(p => p.JavascriptName == name);
```

Comment:
Call `ToList()` here to avoid multiple `enumerations` later. Then change the `.Count()` references.